pandas/_libs/tslib.pyx

```diff
@@ -802,228 +802,33 @@ class Timestamp(_Timestamp):
         return self + other
 
 
-_nat_strings = set(['NaT', 'nat', 'NAT', 'nan', 'NaN', 'NAN'])
 
+# ----------------------------------------------------------------------
+# NaT Construction
+# Here we patch the docstrings of NaT methods to match those of Timestamp
+from tslibs.nattype import (NaT, NaTType,
+                            _make_error_func, _make_nat_func,
+                            _nat_strings)
+from tslibs.nattype cimport _nat_scalar_rules, _checknull_with_nat
 
-def _make_nat_func(func_name, cls):
-    def f(*args, **kwargs):
-        return NaT
-    f.__name__ = func_name
-    f.__doc__ = getattr(cls, func_name).__doc__
-    return f
+# We patch these NaTType methods with the Timestamp docstrings.
+NaTType.astimezone = _make_error_func('astimezone', Timestamp)
```

Comment:
for these 2 you can simply use the ``datetime`` doc-strings no?

Reply:
Timestamp.astimezone is an alias for Timestamp.tz_convert, which has a distinct docstring.